seccomp/mount.go

```diff
@@ -132,6 +135,11 @@ func (m *mountSyscallInfo) process() (*sysResponse, error) {
 func (m *mountSyscallInfo) processProcMount(
 	mip *mountInfoParser) (*sysResponse, error) {
 
+	// Adjust mount attributes attending to process' root path.
+	if err := m.rootAdjust(); err != nil {
```

Comment:
What is rootAdjust() doing? Making the path's absolute? or more ...

Reply:
No, it's not converting the path to absolute (it is already absolute by now -- prior routine takes care of it), rootAdjust() is simply adjusting the "target" mount parameter to account for an offset in the process' root path attribute.